docs/notebooks/keras_wrapper.ipynb

```diff
@@ -0,0 +1,800 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "#### Integration with Keras : Cosine Similarity Task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As an example of using the wrapper with Keras, we try to use the wrapper for word similarity task where we compute the cosine distance as a measure of similarity between the two words."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from keras.engine import Input\n",
+    "from keras.models import Model\n",
+    "from keras.layers.merge import dot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We would use the layer returned by the function `get_embedding_layer` in the Keras model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "wv = model.wv\n",
+    "embedding_layer = wv.get_embedding_layer()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we construct the Keras model. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/chinmaya13/anaconda/lib/python2.7/site-packages/IPython/kernel/__main__.py:7: UserWarning: Update your `Model` call to the Keras 2 API: `Model(outputs=Tensor(\"do..., inputs=[<tf.Tenso...)`\n"
+     ]
+    }
+   ],
+   "source": [
+    "input_a = Input(shape=(1,), dtype='int32', name='input_a')\n",
+    "input_b = Input(shape=(1,), dtype='int32', name='input_b')\n",
+    "embedding_a = embedding_layer(input_a)\n",
+    "embedding_b = embedding_layer(input_b)\n",
+    "similarity = dot([embedding_a, embedding_b], axes=2, normalize=True)\n",
+    "\n",
+    "keras_model = Model(input=[input_a, input_b], output=similarity)\n",
+    "keras_model.compile(optimizer='sgd', loss='mse')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we input the two words which we wish to compare and retrieve the value predicted by the model as the similarity score of the two words. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[[ 0.00596689]]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "word_a = 'graph'\n",
+    "word_b = 'trees'\n",
+    "output = keras_model.predict([np.asarray([model.wv.vocab[word_a].index]), np.asarray([model.wv.vocab[word_b].index])])  # output is the cosine distance between the two words (as a similarity measure)\n",
```

Comment:
The comment would be better moved after the code, on a separate line, to improve readability.